src/appwindow.cpp

```diff
@@ -1302,6 +1300,12 @@ void AppWindow::on_actionFullScreen_toggled(bool checked)
     auto state = windowState();
     state.setFlag(Qt::WindowFullScreen, checked);
     setWindowState(state);
+    if (!SettingsHelper::isOnFullScreenDialogShown())
+    {
+        QMessageBox::information(this, tr("How to exit full-screen"),
```

Comment:
```suggestion
        QMessageBox::information(this, tr("Hint: Exit Full screen mode"),
```

Reply:
I prefer the current one.

Reply:
It's okay if you prefer. I am fine with either of them.